apps/challenges/views.py

```diff
@@ -84,7 +84,7 @@ def challenge_list(request, challenge_host_team_pk):
         return Response(response_data, status=status.HTTP_406_NOT_ACCEPTABLE)
 
     if request.method == 'GET':
-        challenge = Challenge.objects.filter(creator=challenge_host_team, is_disabled=False)
+        challenge = Challenge.objects.filter(creator=challenge_host_team, is_disabled=False).order_by('id')
```

Comment:
Why are we adding the `order_by` using `id`?

Reply:
It removes the Pagination related warnings.
I used [this](https://stackoverflow.com/questions/44033670/python-django-rest-framework-unorderedobjectlistwarning) link as a reference.

Reply:
How adding `order_by id` removes these warnings?

Reply:
The issue is due to a new control for ordering added in Django 1.11 
https://docs.djangoproject.com/en/1.11/topics/pagination/#required-arguments

> 
```
def _check_object_list_is_ordered(self):
        """
        Warn if self.object_list is unordered (typically a QuerySet).
        """
        if hasattr(self.object_list, 'ordered') and not self.object_list.ordered:
            warnings.warn(
                'Pagination may yield inconsistent results with an unordered '
                'object_list: {!r}'.format(self.object_list),
                UnorderedObjectListWarning
            )`

```

> 

so i used the suggested [way](https://docs.djangoproject.com/en/1.11/topics/pagination/#required-arguments) to remove the error .

Reply:
Cool! This looks good. Also, please order by in `descending` order.

Reply:
On changing the `order by` field to `descending` order test for `test_get_challenge_when_mode_is_host` is Failing.
Should I keep it in `ascending`?

Reply:
yeah, keep it ascending in that.

Reply:
I believe you misunderstood me here. We should keep the order `ascending` in the API for which the tests are failing.